app/download/views.py

```diff
@@ -36,6 +36,41 @@ def download_document(service_id, document_id):
     except ValueError:
         return jsonify(error="Invalid decryption key"), 400
 
+    try:
+        download_document_b64(service_id, document_id)
```

Comment:
`download_document_b64()`and `download_document()` are nearly identical, why does one call the other? Are you intending to try to get the file from the store twice?

Reply:
O I might have understood the prompt wrong, let me add the check_scan_files to document_downloand_64 as well no the other way around 🤦🏽